apps/teamsjs-diagnostic-app/src/components/sample/LoggerUtility.tsx

```diff
@@ -0,0 +1,40 @@
+export const captureConsoleLogs = (callback: (log: string) => void) => {
+    const originalConsoleLog = console.log;
+    const originalConsoleError = console.error;
+  
+    console.log = function (...args: any[]) {
+        const currentDate = new Date();
+        const hours = currentDate.getHours().toString().padStart(2, "0");
+        const minutes = currentDate.getMinutes().toString().padStart(2, "0");
+        const seconds = currentDate.getSeconds().toString().padStart(2, "0");
+        const milliseconds = currentDate.getMilliseconds().toString().padStart(3, "0");
+        const month = (currentDate.getMonth() + 1).toString().padStart(2, "0");
+        const day = currentDate.getDate().toString().padStart(2, "0");
+        const year = currentDate.getFullYear().toString().slice(-2);
+        const formattedTimestamp = `[ ${hours}:${minutes}:${seconds}:${milliseconds} ${month}/${day}/${year} ]`;
```

Comment:
Is it a hard requirement for this time format? Can't we use UTC?

Reply:
You could also choose to just log the Date object as is, which would show up in logs like this:

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/18446904/205356f5-eac5-462e-b732-cc17de3170da)